spec/support/initialize_factory_serializers.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.factories.each do |factory|
+  next unless factory.name.in? %i[user]
+
+  serializer_klass = "#{factory.name.capitalize}Serializer"
+
+  Object.const_set(
```

Comment:
Magic code :heart_eyes:

Reply:
Ya, can we document what's happening here please?

Reply:
I _think_ that you're defining a new class on-the-fly here ... is there a clearer way to do it? Even moving the `Class.new` out of the `Object.const_set` might help imo.